qtfred/ui/BackgroundEditor.ui

```diff
@@ -0,0 +1,889 @@
+          <item row="0" column="0">
+           <widget class="QCheckBox" name="noLightingCheckBox">
+            <property name="text">
+             <string>No lighting</string>
+            </property>
+           </widget>
+          </item>
+          <item row="0" column="1">
+           <widget class="QCheckBox" name="transparentCheckBox">
+            <property name="text">
+             <string>Transparent</string>
+            </property>
+           </widget>
+          </item>
+          <item row="0" column="2">
+           <widget class="QCheckBox" name="forceClampCheckBox">
+            <property name="text">
+             <string>Force clamp</string>
+            </property>
+           </widget>
+          </item>
+          <item row="1" column="0">
+           <widget class="QCheckBox" name="noZBufferCheckBox">
+            <property name="text">
+             <string>No z-buffer</string>
+            </property>
+           </widget>
+          </item>
+          <item row="1" column="1">
+           <widget class="QCheckBox" name="NoCullCheckBox">
+            <property name="text">
+             <string>No cull</string>
+            </property>
+           </widget>
+          </item>
+          <item row="1" column="2">
+           <widget class="QCheckBox" name="NoGlowMapsCheckBox">
+            <property name="text">
+             <string>No glow maps</string>
+            </property>
+           </widget>
+          </item>
+         </layout>
+        </item>
+       </layout>
+      </widget>
+     </item>
+     <item>
+      <widget class="QGroupBox" name="groupBox_6">
+       <property name="title">
+        <string>Misc</string>
+       </property>
+       <layout class="QVBoxLayout" name="verticalLayout_2">
+        <item>
+         <widget class="QLabel" name="label_34">
+          <property name="text">
+           <string>Number of stars: 500</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QSlider" name="numStarsSlider">
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QCheckBox" name="subspaceCheckBox">
+          <property name="text">
+           <string>Takes place inside subspace</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <layout class="QHBoxLayout" name="horizontalLayout_20">
+          <item>
+           <widget class="QPushButton" name="envMapButton">
+            <property name="text">
+             <string>Environment Map</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLineEdit" name="envMapEdit"/>
+          </item>
+         </layout>
+        </item>
+       </layout>
+      </widget>
+     </item>
+    </layout>
+   </item>
+  </layout>
+  <zorder></zorder>
```

Comment:
The Qt .ui compiler is complaining about this and the following line.
The warning message is this: `qtfred/ui/BackgroundEditor.ui: Warning: Z-order assignment: '' is not a valid widget.`

Reply:
Weird about the zorder lines, not sure how they got there. Added the SHIFT+I shortcut for Background Editor, I think that's what you were referring to? Were there other shortcuts as well? Although does it make sense to add the shortcut when the dialogs currently don't do anything?